### ISSUE TITLE
duplicate students but same name in Admin page - which to delete?

### ISSUE THREAD

**Issue by techmagpie** [RunestoneInteractive/RunestoneServer#1431]
**What Course are you in**
williamg21 based on thinkcspy

**What Page were you on**
n/a

**What is your username**
mraharrison

**Describe the bug**
Duplicate Student accounts, but removing them through the user interface is almost impossible:

I have managed to register most of my 23 students twice. They have completed a lot of the course already. I want to delete the inactive users, leaving only the accounts they are active on, but in the Admin page the Remove Students feature only shows a list of student names e.g. John Doe, no other fields are shown so I cannot work out which is the inactive student name (johndoe or JohnDoe). 

**Traceback**
n/a

**Additional context**

**Comment by bnmnetp** [comment]
Well, you are not meant to register your students twice, so this is not a scenario I had envisioned.  😄  Did you register them via the csv file method?  How did this happen?

Students themselves can choose to delete an account so if they know which is their active account and which is their duplicate they could log in and delete the duplicate.  But I recognize that is fraught with danger.

In the meantime I have found a way to add the username in parentheses next to their First and Last names.  That change should be live by tomorrow.
